tests/ssl/generate_tls_keys.sh

```diff
@@ -109,7 +109,7 @@ gen_crl() {
         -out "${OUT_DIR}/$signer/crl.pem"
 }
 
-declare -a names=("mysql" "postgresql" "acra-writer" "acra-writer-2" "acra-writer-revoked" "acra-server" "ocsp-responder" "intermediate-ca" "vault" 'acra-client')
+declare -a names=("mysql" "postgresql" "acra-writer" "acra-writer-2" "acra-writer-revoked" "acra-server" "ocsp-responder" "intermediate-ca" "vault" 'acra-client' 'redis')
```

Comment:
It looks a bit inconsistent to use different types of quotes in the array declaration, when quotes actually do not needed at all :)
```suggestion
declare -a names2=(mysql postgresql acra-writer acra-writer-2 
    acra-writer-revoked acra-server ocsp-responder intermediate-ca vault 
    acra-client redis)
```